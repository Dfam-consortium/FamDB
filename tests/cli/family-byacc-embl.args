family
TEST0001
-f
embl
