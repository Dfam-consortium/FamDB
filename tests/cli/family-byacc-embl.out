ID   TEST0001; SV 1; linear; DNA; STD; UNC; 4 BP.
NM   Test family TEST0001
XX
AC   TEST0001;
XX
XX
KW   /.
XX
OS   root
OC   .
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: root
CC        SearchStages: 
CC        BufferStages: 
XX
SQ   Sequence 4 BP; 1 A; 1 C; 1 G; 1 T; 0 other;
     acgt                                                               4
//
